### ISSUE TITLE
Doc or template outdated

### ISSUE THREAD

**Issue by 0xSwapFeeder** [massalabs/massa-docs#103]
Hi,

I encountered the issue where executing the command to set-up the main massa-sc template project I got several issues:
 - some packages where not up to date
 - the src/deploy.ts was outdated (using old version of deploySC() function)
 
Either it's because the version to call from the documentation has not been updated or the template itself is not up to date !

Have a nice day :)

**Comment by qdrn** [comment]
I think it was updated recently (yesterday). Can you have a look again ? cc @massalabs/innovation-dev

**Comment by Thykof** [comment]
Hi, 

I initialized a project with `npx clear-npx-cache && npx @massalabs/sc-project-initializer@dev init my-sc && cd my-sc`.

Then I ran `npm run deploy` and it worked.

I know there is an error with `npm run fmt`, I will fix it soon.

I recommend to run `npm update @massalabs/as-transformer @massalabs/as-types @massalabs/massa-as-sdk @massalabs/massa-sc-compiler @massalabs/massa-sc-deployer @massalabs/massa-web3` just after initializing a project, to make sure to have the latest versions of all pakcages.
